.github/workflows/test-spec.yml

```diff
@@ -36,6 +39,7 @@ jobs:
       OPENSEARCH_DOCKER_REF: ${{ matrix.entry.ref }}
       OPENSEARCH_VERSION: ${{ matrix.entry.version }}
       OPENSEARCH_PASSWORD: ${{ matrix.entry.admin_password || 'myStrongPassword123!' }}
+      OPENSEARCH_EXPERIMENTAL_FEATURES: ${{ matrix.entry.opts || 'null' }}
```

Comment:
This should be `OPENSEARCH_JAVA_OPTS` because it's not just a list of experimental features. 

Does it default to a string "null"? That's not the same thing as undefined. IMO it should read this:

```
OPENSEARCH_JAVA_OPTS: ${{ matrix.entry.opts }}
```

Reply:
will update this